src/tests/unit/tests/DetailsView/reports/components/report-sections/details-section.test.tsx

```diff
@@ -2,13 +2,16 @@
 // Licensed under the MIT License.
 import { shallow } from 'enzyme';
 import * as React from 'react';
-
+import { DateProvider } from '../../../../../../../common/date-provider';
 import { DetailsSection, DetailsSectionProps } from '../../../../../../../DetailsView/reports/components/report-sections/details-section';
 
 describe('DetailsSection', () => {
     it('renders', () => {
+        const scanDate = new Date(Date.UTC(2018, 2, 9, 9, 48));
+        const getUTCStringFromDateStub: typeof DateProvider.getUTCStringFromDate = anyDate => '2018-03-12 11:24 PM UTC';
```

Comment:
shouldn't this be a mock? that way you can assert it's being called with `scanDate`